microprofile-jwt/src/main/java/org/jboss/eap/qe/microprofile/jwt/tools/LogListener.java

```diff
@@ -0,0 +1,62 @@
+package org.jboss.eap.qe.microprofile.jwt.tools;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.io.input.Tailer;
+import org.apache.commons.io.input.TailerListener;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Basic tailer log listener, base on waiting on pattern to appear.
+ */
+@Slf4j
```

Comment:
Why this and not jboss logging ?

Reply:
Just a personal preference. Will move to jboss-logging.